forge-gui/res/cardsfolder/upcoming/two_headed_hellkite.txt

```diff
@@ -0,0 +1,10 @@
+Name:Two-Headed Hellkite
```

Comment:
Just FYI.. it's normally fine to leave "two-headed" in the file name when the card name has the hyphen present. We only really use _ for a space. But it's up to you, not a big deal. Both ways are fine here.

Reply:
Ok will keep in mind !